examples/benchmarks/memory_bw_performance.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""Micro benchmark example for device memory bandwidth performance.
+
+Commands to run:
+  python3 examples/benchmarks/cuda_memory_bw_performance.py
+"""
+
+from superbench.benchmarks import BenchmarkRegistry, Platform
+from superbench.common.utils import logger
+
+if __name__ == '__main__':
+    context = BenchmarkRegistry.create_benchmark_context('mem-bw', platform=Platform.CUDA)
```

Comment:
How to add AMD part for 'mem-bw'?

Reply:
as guoshuai's comment, now we may need to add in a another file and use platform=Platform.ROCM, but may be changed design and unify later